environment.yml

```diff
@@ -0,0 +1,10 @@
+name: IvadoMedEnv
```

Comment:
@dyt811  Could you explain what the purpose of this file is?

Reply:
This environment YML is provided such that conda users can easily instantiate a conda environment while still be able to source the configure file from a single source of truth which is shared across multiple dev/user environment setups such as: 1) CPU Dev, 2) GPU users. 

Conda user can certainly just instantiate their own environment and pip install the requirement as well, just save them a few key strokes.